src/components/AddSample/AddSample.jsx

```diff
@@ -0,0 +1,378 @@
+
+import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+// material ui imports 
+import { makeStyles } from '@material-ui/core/styles';
+import { TextField, 
+  Button, MenuItem,
+  FormHelperText, FormControl, Select, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  inputs: {
+   margin: theme.spacing(2),
+  },
+  formControl: {
+    margin: theme.spacing(2),
+    minWidth: 180,
+  },
+}));
+
+
+function AddSample() {
+  // material ui
+  const classes = useStyles();
+
+  // set stuff so we can use them
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  // get state from redux store
+  const user = useSelector(store => store.user);
+
+  // local states for input fields
+  const [ingredientName, setName] = useState('');
+  const [lotNumber, setLotNumber] = useState('');
+  const [format, setFormat] = useState('');
+  const [ingredientAmount, setAmount] = useState('');
+  const [ingredientUnit, setIngredientUnit] = useState('');
+  const [purity, setPurity] = useState('');
+  const [dateManufactured, setDate] = useState('');
```

Comment:
Ideally, we're using `Date` objects instead of strings here.

```js
// Initial state is current date
const [dateManufactured, setDate] = useState(new Date());
```

Then convert your form input values to dates, too

```js
onChange={(event) => setDateManufactured(new Date(event.target.value))}
```
```